packages/SketchMorph2-Core.package/M2Backend.class/instance/restoreBrushSize.color.isFilling.tool..st

```diff
@@ -1,5 +1,6 @@
 undo
-restoreColor: aColor isFilling: aBoolean tool: aSymbol
+restoreBrushSize: aNumber color: aColor isFilling: aBoolean tool: aSymbol
```

Comment:
Why is the method now called `restoreBrushSize`? It seems to do more than that.

Reply:
It restores the brush size, the color, the filling mode and the tool. Before this push, the method was called `restoreColor:isFilling:tool:`, now it is called `restoreBrushSize:color:isFilling:tool:`. What's wrong with that?

Reply:
Ah, I see, it looked like the method name only wants to change the brush size, but restores everything else as a 'side effect' (still confused by Smalltalk method names 😅)